src/main/java/com/corndel/cashregister/Level5.java

```diff
@@ -26,9 +26,38 @@ public class Level5 {
    * If you want to re-use some functions from other levels, you can do, e.g.
    * Level1.addItem(...) and so on.
    */
-  public static List<Item> transaction(int cost, List<Item> paid, List<Item> drawer) {
-    // TODO
-    return null;
+
+    public static List<Item> transaction(int cost, List<Item> paid, List<Item> drawer) {
+
+
+      // Step 1: Add paid items to the drawer
```

Comment:
Good work on getting a working solution. When you have a chance, see how you can introduce some of the other methods you created in the previous exercises.